x/bank/client/cli/tx.go

```diff
@@ -83,13 +83,15 @@ When using '--dry-run' a key name cannot be used, only a bech32 address.
 // For a better UX this command is limited to send funds from one account to two or more accounts.
 func NewMultiSendTxCmd(ac address.Codec) *cobra.Command {
 	cmd := &cobra.Command{
-		Use:   "multi-send [from_key_or_address] [to_address_1, to_address_2, ...] [amount]",
+		Use:   "multi-send [from_key_or_address] [to_address_1 to_address_2 ...] [amount]",
 		Short: "Send funds from one account to two or more accounts.",
 		Long: `Send funds from one account to two or more accounts.
 By default, sends the [amount] to each address of the list.
 Using the '--split' flag, the [amount] is split equally between the addresses.
-Note, the '--from' flag is ignored as it is implied from [from_key_or_address].
+Note, the '--from' flag is ignored as it is implied from [from_key_or_address] and 
+separate addresses with space.
 When using '--dry-run' a key name cannot be used, only a bech32 address.
+Example: ./simd tx bank multi-send cosmos1... cosmos1... cosmos1... cosmos1... 10stake
```

Comment:
Can we use the `Example:` field in the `cobra.Command` struct.

Reply:
ditto 👍

Reply:
+1